projects/igniteui-angular/src/lib/grids/headers/grid-header.component.ts

```diff
@@ -276,6 +276,13 @@ export class IgxGridHeaderComponent implements DoCheck, OnDestroy {
         this.sortDirection = expr ? expr.dir : SortingDirection.None;
     }
 
+    protected isAdvancedFilterApplied() {
+        if(!this.grid.advancedFilteringExpressionsTree) {
+            return false;
+        }
+        return !!this.grid.advancedFilteringExpressionsTree.find(this.column.field);
```

Comment:
`some` can be used instead of `find` and this way the value would not need to be converted to a boolean

Reply:
this is not an array, advancedFilteringExpressionsTree is of type IFilteringExpressionsTree so it exposes only `find` and `findIndex` methods